pyinjective/client/chain/grpc/chain_grpc_exchange_api.py

```diff
@@ -575,5 +575,52 @@ async def fetch_market_atomic_execution_fee_multiplier(
 
         return response
 
+    async def fetch_active_stake_grant(self, grantee: str) -> Dict[str, Any]:
+        request = exchange_query_pb.QueryActiveStakeGrantRequest(grantee=grantee)
+        response = await self._execute_call(call=self._stub.ActiveStakeGrant, request=request)
+
+        return response
+
+    async def fetch_grant_authorization(
+        self,
+        granter: str,
+        grantee: str,
+    ) -> Dict[str, Any]:
+        request = exchange_query_pb.QueryGrantAuthorizationRequest(
+            granter=granter,
+            grantee=grantee,
+        )
+        response = await self._execute_call(call=self._stub.GrantAuthorization, request=request)
+
+        return response
+
+    async def fetch_grant_authorizations(self, granter: str) -> Dict[str, Any]:
+        request = exchange_query_pb.QueryGrantAuthorizationsRequest(granter=granter)
+        response = await self._execute_call(call=self._stub.GrantAuthorizations, request=request)
+
+        return response
+
+    async def fetch_l3_derivative_orderbook(
```

Comment:
what's the l3 derivative orderbook?

Reply:
This is a new endpoint in the chain exchange module